.github/workflows/build_test-PR-templates.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and Test Template tests on Pull Requests
```

Comment:
This seems like an odd name for the workflow. Are we testing the tests? Or testing the PR?

Reply:
we are testing the product when a pull request is created